### DISCUSSION TITLE
Laser cutter and ventilation

### DISCUSSION THREAD

**Discussion by Lpeterson157** [Snapmaker/Luban#1023] [Q&A]
I have the Snapmaker 2.0 A350 with the enclosure, and while I have done a few 3D prints, I have yet to try the laser cutter. I live in an apartment building, and am concerned that it will smell like the building is on fire.

We have an Epilog 40W Fusion at my school, and even with the air scrubber on, it often smells like burning wood. I do recognize that the Epilog has a more powerful laser, however, so I was wondering what the experience of the group is. 

If I get a longer hose, and run it out the window with a setup for venting similar to a portable air conditioner, rather than just pointing it through the nearby door (which would then have open space above and below for air blowback), would that be sufficient to prevent disturbing my neighbors?

I have considered venting into a duct that vents my stove to the outside, but I'm not sure if I'd need a fan at the end to pull the air through, so it didn't just end up in the kitchen...

Or is there another solution that might work?
Any ETA on Snapmaker's air scrubber, or what the system would look like?

Thanks for any help any of you can give on this matter.

**Comment by RobertMela** [comment]
I have been running my A350 in the basement and the laser fumes migrate upstairs.  I just moved it to a second floor 3ft deep closet.

However, the attic is directly above the closet and I will certainly be installing an exhaust blower and vent to the attic.

Venting these fumes outside is a mandatory, not an option.
